packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -55,20 +50,17 @@ async function runPipelineLoadStep(
     if (!step.load) {
       continue;
     }
-    const pluginLogger =
-      pluginLoggers[step.name] || createLogger({name: step.name, level: logLevel});
-    pluginLoggers[step.name] = pluginLogger;
 
     try {
       const debugPath = path.relative(process.cwd(), srcPath);
-      pluginLogger.debug(`load() starting: [${debugPath}]`);
+      logger.debug(`load() starting… [${debugPath}]`, {name: step.name});
       const result = await step.load({
         fileExt: srcExt,
         filePath: srcPath,
         isDev,
         isHmrEnabled,
       });
-      pluginLogger.debug(`load() successful [${debugPath}]`);
+      logger.debug(`✔ load() success [${debugPath}]`, {name: step.name});
```

Comment:
Minor change: when I was messing with the `snowpack build --debug` output, the starting and success needed some differentiation